scripts/updateReadme.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let parseComments = require('parse-comments');
```

Comment:
Can we have these and other variables that do not change be declared with `const`